src/Microsoft.AspNetCore.Mvc.RazorPages/AutoValidateAntiforgeryPageApplicationModelProvider.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+
+namespace Microsoft.AspNetCore.Mvc.RazorPages
+{
+    internal class AutoValidateAntiforgeryPageApplicationModelProvider : IPageApplicationModelProvider
+    {
+        // The order is set to execute after the DefaultPageApplicationModelProvider.
+        public int Order => -1000 + 10;
```

Comment:
Relying on the hardcoded values for orders isn't good in terms of maintaining relative orders. Why won't we define constants (or `static readonly` fields) to reference instead?

Reply:
For one, this code was moved as is (I changed the namespace from `.Internal` -> `internal class`) which git really doesn't seem to track as a rename. That said, we only ever have one constant for all our providers - viz the default is `-1000`. We really don't have any significant numbers outside of it. There really isn't a lot of value in turning that one digit in to a constant.